releases module cannot be found and would cause un-needed d'loading
diff -ur Fabric-1.8.3.orig/docs/conf.py Fabric-1.8.3/docs/conf.py
--- docs/conf.py	2014-02-15 01:28:56.000000000 +0800
+++ docs/conf.py	2014-04-09 22:55:42.851674861 +0800
@@ -26,7 +26,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'releases']
+extensions = ['sphinx.ext.autodoc']
 
 # 'releases' (changelog) settings
 releases_issue_uri = "https://github.com/fabric/fabric/issues/%s"
